### PR TITLE
Refactor: beam editor tags

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- improve beam editor add tags flow and design

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

**Comment by themonster2015** [comment]
After adding some tags and going back to the main beam editor, if I wanted to go back and delete some tags, I wasn't able to click on the Save button because it remained disabled, but after clicking on Cancel and going back to the main editor, the tags actually got removed. It was confusing.

**Comment by themonster2015** [comment]
The delete flow works fine now 👍

File: ui/apps/akasha/src/extensions/beam-editor/beam-editor.tsx [hunk 1]
```diff
@@ -66,6 +70,18 @@ export const BeamEditor: React.FC = () => {
     }
   }, [blocksInUse]);
 
+  React.useEffect(() => {
```
Comment (quininez) [inline ui/apps/akasha/src/extensions/beam-editor/beam-editor.tsx]: why not do the checks from this useEffect in the addTag function?

Comment (josenriagu) [reply ui/apps/akasha/src/extensions/beam-editor/beam-editor.tsx]: Alright. Let me check also in the keyup handler

**Review by quininez — APPROVED** [review]

**Pull request merged**